src/app/query/ProductList.query.js

```diff
@@ -426,6 +426,37 @@ class ProductListQuery {
             .addField(filterItems);
     }
 
+    /**
+     * Prepare review summary field
+     * @returns {Field}
+     * @private
+     */
+    _prepareReviewSummaryField() {
+        return new Field('review_summary')
+            .addField('rating_summary')
+            .addField('review_count');
+    }
+
+    /**
+     * Prepare review summary field
+     * @returns {Field}
+     * @private
+     */
+    _prepareReviewsField() {
+        const ratingVotes = new Field('rating_votes')
```

Comment:
May I assume this is only needed on PDP? Then why to load it for each request?

Reply:
You are correct, reviews field is only used for review list display on PDP. Could you please where can this be moved in order to load the field only on PDP?